src/commands/deployment/correlate.ts

```diff
@@ -25,11 +25,18 @@ export class DeploymentCorrelateCommand extends Command {
       This command will correlate the pipeline with a GitOps CD deployment.\n
       See README for additional details.
     `,
-    examples: [['Correlate an Argo CD deployment', 'datadog-ci deployment correlate --provider argocd']],
+    examples: [
+      ['Correlate an Argo CD deployment', 'datadog-ci deployment correlate --provider argocd'],
+      [
+        'Correlate ArgoCD deployment manually',
+        'datadog-ci deployment correlate --provider argocd --config-repo https://github.com/my-manifests-repo --config-shas 92eb0db6926aaf51b9fb223895b6d8d1c0ff1ff4',
```

Comment:
could we add an example where multiple shas are passed? or modify this one to have multiple. Some devs might not know how to pass multiple SHAs (is it by separating them by comma? Is it passing `--config-shas` multiple times?)